### ISSUE TITLE
Press Button on Reader to Connect Message

### ISSUE THREAD

**Issue by maulikpat** [square/reader-sdk-ios-quickstart#8]
I am getting below message on top of the view.

![img_6567bfab45e1-1](https://user-images.githubusercontent.com/2304770/49068070-c9635780-f24b-11e8-91b4-95fa097f356b.jpeg)

I have already attached Magstripe Reader via Audio Jack. 

I am still getting message to connect. Is it a bug ?

**Comment by StephenJosey** [comment]
That message is regarding the Square Contactless Reader (https://squareup.com/shop/hardware/us/en/products/chip-credit-card-reader-with-nfc). If you've previously connected a reader to your device, this message will be displayed to let you know it isn't currently connected. If you want to forget the reader (to remove the message), you can do so by opening the POS application, clicking the three "option" lines in the top-left corner, choosing "Settings", choosing "Card Readers" and then click on a reader and there will be an option to "Forget this reader".

In other words, this should not affect your magstripe reader - it should still work with this message. Please let me know if that is not the case. Thanks!
